README.md

```diff
@@ -1,2 +1,124 @@
-# CodeReview
-This Repository is only for learning how to review a project in github.
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  
+  
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Mobile First Project] <a name="about-project"></a>
+
+I am starting this project to Creat my own Portfolio.
+
+## 🛠 Built With <a name="built-with"></a>
```

Comment:
Pleas costomize your Readme file according to your project. Because Your Readme file tells everything about your project.